src/main/java/core/basesyntax/Colori.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Colori {
```

Comment:
The name of the enum `Colori` is not written in English. It is better to name it `Color`.